netdiff/parsers/base.py

```diff
@@ -74,7 +75,12 @@ def _to_python(self, data):
             try:
                 return json.loads(data)
             except ValueError:
-                raise NetParserException('Could not decode JSON data')
```

Comment:
Here I think the best thing is to raise another type of exception (NetParserJsonException), and intercept this exception in the OlsrParser.

Don't worry though, I'll take care of implementing and adding tests